pd/src/pending_block.rs

```diff
@@ -43,31 +41,55 @@ pub struct PendingBlock {
     reward_counter: u64,
     /// Records pending state changes to validators.
     pub validator_state_changes: BTreeMap<IdentityKey, ValidatorState>,
+    /// Records all the quarantined inputs/outputs from this block.
```

Comment:
The pending block is getting pretty heavy, we probably want to encapsulate the unbonding logic and data